src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MethodTypeInference.cs

```diff
@@ -2498,80 +2498,6 @@ private bool Fix(int iParam, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
             return best;
         }
 
-        internal static TypeWithAnnotations Merge(TypeWithAnnotations first, TypeWithAnnotations second, VarianceKind variance, ConversionsBase conversions)
-        {
-            var merged = MergeTupleNames(MergeDynamic(first, second, conversions.CorLibrary), second);
-            if (!conversions.IncludeNullability)
-            {
-                // https://github.com/dotnet/roslyn/issues/30534: Should preserve
-                // distinct "not computed" state from initial binding.
```

Comment:
does this comment need to be preserved somewhere?

Reply:
No, because the code now does as the comment says it should because the line below has been deleted.  It now preserves the state from initial binding.

---
In reply to: [332163168](https://github.com/dotnet/roslyn/pull/39069#discussion_r332163168) [](ancestors = 332163168)